src/toil/batchSystems/kubernetes.py

```diff
@@ -88,6 +101,12 @@ def supportsAutoDeployment(cls):
     @classmethod
     def supportsWorkerCleanup(cls):
         return False
+
+    def try_kubernetes(self, method, *args, **kwargs):
```

Comment:
This function needs a docstring that explains why we have it and admonishes everyone to use it instead of making calls directly. It could also have an example of how to use it in conjunction with `self._api`.

It also might be worth renaming it to `_try_kubernetes`, since it shouldn't really be called outside the class. Or, since this is the Kubernetes batch system anyway, we could call it `_retry` or `_retry_call` or something.

And I think it should move down. It shouldn't come before `__init__`. I would put it right after `_api`.